diagnostic.md

```diff
@@ -3,36 +3,47 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+- Ember provides a complete solution when it comes to a client side app.
+- The app runs within the users browser and is self contained, including everything it needs.
+- Ember is meant to be completely backward compatible so that past, or even outdated, innovations
+  are maintained.
 ```
 
 What is the templating system used in Ember? How it is related to
 handlebars?
 
 ```sh
-# your answer here
+Ember uses Handlebars.  Handlebars is a templating engine that is also
```

Comment:
Yes. Ember's version of handlebars comes with some added enhancements: built-in helpers like the `link-to` tag, as well as the ability to plug in components and other templates as needed.
.5/1